Content.Client/Power/PowerMonitoringWindow.xaml.Widgets.cs

```diff
@@ -103,6 +103,29 @@ public void UpdateWindowEntryButton(NetEntity netEntity, PowerMonitoringButton b
 
         // Update power value
         button.PowerValue.Text = Loc.GetString("power-monitoring-window-value", ("value", entry.PowerValue));
+
+        // Update battery level if applicable
+        if (entry.BatteryLevel != null)
+        {
+            var batteryPercentage = (int) (entry.BatteryLevel.Value * 100);
+            button.BatteryLevel.Text = $"{batteryPercentage}%";
+            button.BatteryLevel.Visible = true;
+
+            // Set battery level color based on percentage
+            Color color;
+            if (batteryPercentage > 75)
```

Comment:
pretty sure it would be cleaner a la

```
var color = batteryPercentage switch
{
    > 75 => Color.Green,
    > 25 => Color.Yellow,
    _ => Color.Red
};
```

Reply:
Fixed